src/axom/core/CMakeLists.txt

```diff
@@ -97,10 +97,7 @@ set( core_depends fmt )
 blt_list_append( TO core_depends ELEMENTS camp IF CAMP_FOUND )
 blt_list_append( TO core_depends ELEMENTS umpire IF UMPIRE_FOUND )
 blt_list_append( TO core_depends ELEMENTS RAJA IF RAJA_FOUND )
-list(APPEND core_depends ${axom_device_depends})
```

Comment:
Why is it that we can do away with this line?

Reply:
Oh, I see--- that is what `axom_add_executable` and `axom_add_library` do for us.